### ISSUE TITLE
Change the "Change was Successfull-bar" to a Tooltip/Overlay

### ISSUE THREAD

**Issue by xF4m3** [TechnicPack/TechnicSolder#239]
If you change a version inside a build, you get notified that the "change was successful" (with this little bar at the top of your build) this shifts the whole list down and if you are editing the 100th mod you can't even see this message/bar , so it would be nice if it would be changed to a tooltip next to your mouse so that the whole list doesnt get shifted :) or maybe just make the thing that pops up at the top be a Overlay or something so the whole lsit stays where it is. I is irritating when the whole list is moving down and then up again.

**Comment by fcpwiz** [comment]
Good suggestion! Similar to #144

**Comment by xF4m3** [comment]
ohh ok yes that would be pretty similar, didnt saw this one

**Comment by fcpwiz** [comment]
Yeah. I find it really annoying when you are updating a bunch of mods, and it keeps shifting up and messing you up.

**Comment by Brycey92** [comment]
I also think this would be a very helpful change, as I have messed up version changes countless times due to all the shifting.

**Comment by TheDoctorsLife** [comment]
I like this, can we please have it?
